browser/brave_ads/BUILD.gn

```diff
@@ -12,6 +12,18 @@ source_set("brave_ads") {
     "ads_tab_helper.h",
   ]
 
+  if (is_android) {
```

Comment:
these files should be have an ads_enabled guard

Reply:
actually we should just get rid of them, but the replacement brave_ads_host file in brave/browser/brave_ads should be behind the brave ads buildflag

Reply:
Fixed, buildflag was removed here: https://github.com/brave/brave-core/pull/9769